go/keymanager/api/api.go

```diff
@@ -27,9 +27,12 @@ const (
 )
 
 var (
+	// ErrInvalidArgument is the error returned on malformed arguments.
+	ErrInvalidArgument = errors.New(ModuleName, 1, "keymanager: invalid argument")
+
 	// ErrNoSuchStatus is the error returned when a key manager status does not
 	// exist.
-	ErrNoSuchStatus = errors.New(ModuleName, 1, "keymanager: no such status")
+	ErrNoSuchStatus = errors.New(ModuleName, 2, "keymanager: no such status")
```

Comment:
Adding `ErrInvalidArgument` with not `1` as the error code would be a breaking change anyway, so I updated this one to `2`. That way code `1` is consistently used for `ErrInvalidArgument` errors.